lnclient/lnd/lnd.go

```diff
@@ -250,12 +250,20 @@ func (svc *LNDService) subscribeChannelEvents(ctx context.Context) {
 						"reason":               closureReason,
 					}).Info("Channel closed")
 
+					var fundingTxId string
+					channelPoint, _ := svc.parseChannelPoint(update.ClosedChannel.ChannelPoint)
+					if channelPoint != nil {
+						fundingTxId = channelPoint.GetFundingTxidStr()
+					}
+
 					svc.eventPublisher.Publish(&events.Event{
 						Event: "nwc_channel_closed",
 						Properties: map[string]interface{}{
 							"counterparty_node_id": counterpartyNodeId,
 							"reason":               closureReason,
 							"node_type":            config.LNDBackendType,
+							"funding_tx_id":        fundingTxId,
+							"incoming_balance":     update.ClosedChannel.SettledBalance,
```

Comment:
This is not _incoming_ since in LND this triggers after close has finished, so the balance is already deposited in the users wallet by this point.

Reply:
Hmm, the intention of this event is to notify the user immediately when their channel begins closing so they know and can instantly do something (e.g. open a new channel if they want). Is there any event for this?

e.g. if this is a force close then this event wouldn't happen for days?

Reply:
There's `ChannelEventUpdate_InactiveChannel` which triggers when the channel is closed, will try to see if that can work here.

Reply:
It only gives the chan point, so we can get funding tx id but not the balance. And I tried to see if we can get the closed channel info using `GetChanInfo` but it says `edge not found` because it has been closed...

Reply:
Also observed that this `ChannelEventUpdate_InactiveChannel` doesn't fire every time (on channel close)